system/mount.py

```diff
@@ -252,6 +252,43 @@ def umount(module, **kwargs):
     else:
         return rc, out+err
 
+def ismounted(module, target, source):
+    """ check is target-source mount point exist (including bind) """
+    mounted = False
+
+    # bind mounts are not recognized correctly by `os.path.ismount` method
+    cmd = 'findmnt -rn %s' % target
+    rc, out, err = module.run_command(cmd)
+
+    allmounts = out.split('\n')
+
+    for mounts in allmounts[:-1]:
+       arguments = mounts.split()
+       sourcePath = re.compile('\[(.*)\]').search(arguments[1]).group(1)
+
+       if source == sourcePath:
+           mounted = True
+
+    """ check is path is mounted (including bind) """
+    mounted = False
```

Comment:
@jupeter, this statement make all previous function part useless since its result is actually ignored.